README.md

```diff
@@ -127,13 +127,7 @@ docker run \
 
 After rotating the TLS assets, there are some **important** manual steps that should be done:
 
-1. Update locally the `CA` certificate for the API server of this cluster
```

Comment:
Those were already in the kube-upgrade branch

Reply:
I am going to decline this PR. Thank you @sbuliarca